lib/tasks/site_customization_pages.rake

```diff
@@ -0,0 +1,15 @@
+namespace :site_customization_pages do
+  desc "Recover background_color and font_color defined in deploy.yml"
+  task update_color_fields_values_from_deploy_yml: :environment do
+    SiteCustomization::Page.all.each do |page|
+      background_color = Rails.configuration.deploy.dig("custom_pages", page.slug, "bgcolor")
+      font_color = Rails.configuration.deploy.dig("custom_pages", page.slug, "color") || "#FFF"
```

Comment:
Why `|| "#FFF"`? Is there a case where the background color is defined but the font color isn't defined? :thinking: